app/Jobs/ImportTurboVotePosts.php

```diff
@@ -390,7 +390,15 @@ private function getCSVRecords($filepath)
         $file = Storage::get($filepath);
         $csv = Reader::createFromString($file);
         $csv->setHeaderOffset(0);
-        $records = $csv->getRecords();
+        // $csv = str_replace("\r","\n", $csv);
```

Comment:
@chloealee Can you try doing the string replacement above, before putting it into the CSV parser? I think you might be running into trouble since `$csv` is a CSV reader object down below.

```php
$file = Storage::get($filepath);
$file = str_replace("\r", "\n", $file);

$csv = Reader::createFromString($file);
```

Reply:
omg you are so right, that worked! Thank you @DFurnes !